tests/Doctrine/Tests/ORM/Functional/Ticket/GH7864Test.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+
+use Doctrine\Tests\Models\Tweet\Tweet;
+use Doctrine\Tests\Models\Tweet\User;
+use Doctrine\Tests\OrmFunctionalTestCase;
+use function array_values;
+
+class GH7864Test extends OrmFunctionalTestCase
+{
+    protected function setUp() : void
+    {
+        $this->useModelSet('tweet');
```

Comment:
We're avoiding using model sets and putting all necessary classes in the test file. That helps a lot to keep the SUT more explicit. Would you please do the same here?